### ISSUE TITLE
Create support CPF

### ISSUE THREAD

**Issue by guilhermeepaixaoo** [nodeca/charlatan#14]
Create feature for support in CPF, a document for people registered from Brazil. :credit_card:

**Comment by puzrin** [comment]
Could you rephrase? Not sure i understand the goal of this ticket.

**Comment by guilhermeepaixaoo** [comment]
Sorry, my English is not very good. :sleepy: 

So, I think about create feature for generate CPF valid randomly.

I was using the Charlatan and need to generate CPF valid for tests.

**Comment by puzrin** [comment]
This package mirrors ruby's [Faker](https://github.com/stympy/faker). I think it's better to post your request there - we should not diverge api.

Also i see no problem is you extend charlatan locally as you wish.

**Comment by guilhermeepaixaoo** [comment]
Ok :smile:

**Comment by puzrin** [comment]
Closing?
